lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -129,7 +129,14 @@ def stop_repmgrd
     end
 
     def node_number_valid?
-      rec = record_for_node_number
+      begin
+        rec = record_for_node_number
+      rescue PG::Error => e
```

Comment:
Nothing else in this method will raise a PG error. I would rather still have this at the bottom of the method.